pkg/steps/release/import_release.go

```diff
@@ -188,14 +160,32 @@ func (s *importReleaseStep) run(ctx context.Context) error {
 					Type: imagev1.LocalTagReferencePolicy,
 				},
 				From: &coreapi.ObjectReference{
-					Kind: "DockerImage",
-					Name: cliImage,
+					Kind: "ImageStreamTag",
+					// Hardcode the image to an x86 one, so this works for multi-arch.
+					Name:      "4.9:cli-artifacts",
+					Namespace: "ocp",
```

Comment:
This would be incorrect in some jobs - you can't make `stable:cli` not equal to the version of the release (a 4.6 cluster can't use a 4.9 cli).

If the release image is not the current platform arch, I'd suggest importing the version that matches that release to a separate tag and trying to use that (if it exists) in other ci-operator code.

Reply:
Yeah, this is my comment above - we should be able to change it trivially to use some other intermediate target.